README.md

```diff
@@ -1,2 +1,150 @@
-# OOP--School--Library
-school library app
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [OOP school library] <a name="about-project"></a>
+
+**OOP school library** The "OOP School Library" project is an initial step in building a school library app using object-oriented programming in Ruby. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Language</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **Rubocop implemented**
+- **Implement enumerable module**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+- To clone or fork this repo
+- Ruby installed in your machine
+- To modify this project you'll need a text editor
+
+For more information on how to install ruby on your machine:
+- <a href="https://www.ruby-lang.org/en/documentation/installation/">How to install ruby</a>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd project-folder
+  git clone https://github.com/ANTOINE1128/OOP--School--Library.git
+```
+
+For more information on how to clone or fork a repository:
+- <a href="https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository">How to clone a repo</a>
+- <a href="https://docs.github.com/en/get-started/quickstart/fork-a-repo">How to fork a repo</a>
+
+
+### Install
+
+To install dependencies, the following command must be run in the terminal:
+
+```sh
+  cd OOP--School--Library
+  rubocop
+```
+
+### Usage
+
+In the project directory, you can run:
+
+```sh
+  irb -r ./index.rb
+```
+
+### Run tests
+
+No tests available
+
+### Deployment
+
+No deployment of this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Antoine Makdessy**
+
+- GitHub: [@ANTOINE1128](https://github.com/ANTOINE1128)
+- LinkedIn: [Antoine (Tony) Makdessy](https://www.linkedin.com/in/antoine-makdessy/)
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+Future changes:
+- No changes
+
```

Comment:
- [ ] To make your README looks more professional and descriptive, I think it would be better if you update the current content of the above section to something like this below:
```
## 🔭 Future Features <a name="future-features"></a>

Future changes:
- Add Spec Tests
- Add Basic UI
- Etc